src/main/java/pl/javastart/task/Room.java

```diff
@@ -0,0 +1,53 @@
+package pl.javastart.task;
+
+public class Room {
+
+    private boolean airConditioned;
+    private double roomSize;
+    private double roomTemperature;
+    private double minTemperature;
+
+    public Room(boolean airConditioned, double roomSize, double roomTemperature, double minTemperature) {
+        this.airConditioned = airConditioned;
+        this.roomSize = roomSize;
+        this.roomTemperature = roomTemperature;
+        this.minTemperature = minTemperature;
+    }
+
+    boolean lowerTemperature() {
+        if (airConditioned && roomTemperature > minTemperature) {
+            roomTemperature--;
```

Comment:
Jeżeli `roomTemperature` będzie np. 19.5, a minTemperature to 19, to temperatura spadnie do 18.5, a powinna się zatrzymać na 19.